libs/components/src/lib/checkbox/checkbox.scss

```
@@ -34,11 +36,19 @@ $gutter: 4px;
 	@supports (user-select: none) {
 		user-select: none;
 	}
+
+	&:not(.disabled) {
```

what about readonly?
```suggestion
	&:not(.disabled, .readonly) {
```

---

Readonly is still interactive